custom_components/tahoma/__init__.py

```diff
@@ -76,8 +77,12 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     username = entry.data.get(CONF_USERNAME)
     password = entry.data.get(CONF_PASSWORD)
 
+    session = aiohttp_client.async_create_clientsession(
+        hass, cookie_jar=CookieJar(unsafe=True)
```

Comment:
you can remove cookie_jar parameter. It's useless in our case.

Reply:
I tried, however it doesn't work anymore when I remove this.
https://github.com/iMicknl/ha-tahoma/pull/237#discussion_r478339535

Reply:
That's weird. So I'm fine with this PR.